### ISSUE TITLE
data?

### ISSUE THREAD

**Issue by maximilianh** [bmschmidt/pubmed-explorer#64]
Hi, impressive viz!

I'd like to load the data into our own javascript dim reduction browser, https://cells.ucsc.edu, to see if that's a lot slower than your deep scatter plot (probably it is). Do you have the object coordinates somewhere? And some meta data for the data points, e.g. title, authors, journal, year ?

**Comment by bmschmidt** [comment]
@maximilianh of course! You can download all this data from https://zenodo.org/record/7849020. If anything appears to be amiss or you have any other questions (I'm just right now adding some selection logic into deepscatter that might make it work well for some of your purposes!) please reach out!
